### ISSUE TITLE
refactor: remove notion of cheminformatics and use scikit-learn convention for features and labels

### ISSUE THREAD

**Issue by cvigilv** [cvigilv/SimSpread.jl#18]
As of today, the package has a lot of notion related to cheminformatics and the source of development of this package. We use DT, DD and DF for labels, features and processed features. 

This issue intends to find and change the original naming convention to a more sklearn/ML/MLJ.jl language, making use of `y` and `yhat` for labels and predicted labels and `X` for features. From this starting point, additional symbols can be used to clarify the different steps of the complete algorithm as proposed in the original publication.
